Cargo.toml

```diff
@@ -9,4 +9,6 @@ members = [
 resolver = "2"
 
 [profile.release]
-debug = 2
+debug = 0  # upstream has '2' for some reason
+lto = true  # reduces bin size; also slows link-time, but we don't build often
+strip = true  # automatically strip symbols from the binary.
```

Comment:
Hm I think we shouldn't strip binaries by default. Our other binaries aren't stripped because it helps with debugging.

Reply:
Yeh this isn't necessary if LTO is doing most of the optimization work.